src/main/webapp/index.html

```diff
@@ -26,10 +26,12 @@
     <nav>
       <ul id="navigation">
         <li><a href="/">Home</a></li>
+	<li><a href="/feed.html">Feed</a></li>
```

Comment:
Note: Spacing looks incorrect on github diffs because of the size of the '+' character. If you go into "View file", the spacing is correct.